include/aspect/simulator.h

```diff
@@ -1878,6 +1878,9 @@ namespace aspect
 
       mutable TimerOutput                 computing_timer;
 
+      Timer walltime;
+      double walltime_from_resume;
```

Comment:
Can you add a comment to these variables? Especially, what walltime_from_resume is? Note that this is not the time from the last resume but the total time elapsed up to the last snapshot.
Maybe we should rename the variable to reflect that?

Reply:
Sure. Yes, good point I changed the variable to total_walltime_from_last_snapshot